From 69a423fb4fb62d60625d7c329dd5c57cc735a045 Mon Sep 17 00:00:00 2001
From: Konrad Leszczynski <konrad.leszczynski@intel.com>
Date: Tue, 15 Dec 2015 14:12:28 +0000
Subject: [PATCH 12/12] defconfig: add flags for USB MIDI support

This patch adds the necessart flags to the x86_64_defconfig,
which enable the USB MIDI functionality.

Change-Id: Ic0aca2c03db2d5798bf6bbd4979ed29f90e252a3
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-3355
Signed-off-by: Konrad Leszczynski <konrad.leszczynski@intel.com>
Reviewed-on: https://android.intel.com:443/448874
---
 arch/x86/configs/x86_64_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index a344be2..5392717 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -3153,6 +3153,7 @@ CONFIG_USB_F_ACM=y
 CONFIG_USB_U_SERIAL=y
 CONFIG_USB_F_MASS_STORAGE=y
 CONFIG_USB_F_FS=y
+CONFIG_USB_F_MIDI=y
 # CONFIG_USB_CONFIGFS is not set
 # CONFIG_USB_ZERO is not set
 # CONFIG_USB_AUDIO is not set
-- 
1.9.1

